src/library-authoring/data/api.ts

```diff
@@ -128,25 +128,23 @@ export interface UpdateXBlockFieldsRequest {
 }
 
 /**
- * Fetch block types of a library
+ * Fetch the list of XBlock types that can be added to this library
  */
-export async function getLibraryBlockTypes(libraryId?: string): Promise<LibraryBlockType[]> {
+export async function getLibraryBlockTypes(libraryId: string): Promise<LibraryBlockType[]> {
   if (!libraryId) {
-    throw new Error('libraryId is required');
+    throw new Error('The current API for block types requires a libraryId.');
```

Comment:
Hi @bradenmacdonald! This was more of a type guard than actual validation. I think it is safe to remove this, as it is handled in the hook that calls this function.

Reply:
Sure, removed :)